content/en/docs/ops/diagnostic-tools/istioctl-analyze/index.md

```diff
@@ -178,3 +178,41 @@ You can enable this feature with:
 {{< text bash >}}
 $ istioctl manifest apply --set values.galley.enableAnalysis=true
 {{< /text >}}
+
+### Ignoring specific analyzer messages via CLI
+
+Sometimes you might find it useful to hide or ignore analyzer messages in certain cases. For example, imagine a situation where a message is emitted about a resource you don't have permissions to update:
+
+{{< text bash >}}
+$ istioctl analyze -k --all-namespaces
+Warn [IST0102] (Namespace frod) The namespace is not enabled for Istio injection. Run 'kubectl label namespace frod istio-injection=enabled' to enable it, or 'kubectl label namespace frod istio-injection=disabled' to explicitly mark it as not needing injection
+Error: Analyzers found issues.
+See https://istio.io/docs/reference/config/analysis for more information about causes and resolutions.
+{{< /text >}}
+
+Because you don't have permissions to update the namespace, you cannot resolve the message by annotating the namespace. Instead, you can direct `istioctl analyze` to suppress the above message on the resource:
+
+{{< text bash >}}
+$ istioctl analyze -k --all-namespaces --suppress "IST0102=Namespace frod"
+✔ No validation issues found.
+{{< /text >}}
+
+The syntax used for suppression is the same syntax used throughout `istioctl` when referring to resources: '\<kind> \<namespace>.\<name>', or just '\<kind> \<name>' for cluster-scoped resources like `Namespace`. If you want to suppress multiple objects, you can either repeat the `--suppress` argument or use wildcards:
```

Comment:
Isn't in `<name>.<namespace>`, not `<namespace>.<name>`?

Also, can your syntax examples use backticks, avoiding the need to escape `<`?

Reply:
Yes, it is. 🤦‍♂  Fixed.

I can use backticks - originally that's what I used but I was worried it was confusing to describe syntax with that.